ddtrace/contrib/botocore/patch.py

```diff
@@ -467,10 +495,37 @@ def patched_api_call(original_func, instance, args, kwargs):
         span.set_tag(ANALYTICS_SAMPLE_RATE_KEY, config.botocore.get_analytics_sample_rate())
 
         try:
-            result = original_func(*args, **kwargs)
-            _set_response_metadata_tags(span, result)
+            if endpoint_name == "sqs" and operation == "ReceiveMessage" and config._data_streams_enabled:
+                queue_url = params["QueueUrl"]
+                queue_name = queue_url[queue_url.rfind("/") + 1 :]
+
+                if "MessageAttributeNames" not in args[1]:
```

Comment:
There's no guarantee that args[1] will always exist. The parameter could be passed a key word argument. 

We should use `get_argument_value(args, kwargs, 1, "<insert_kwarg_name_here>")`.